test/integration/sanitation-test.js

```diff
@@ -494,7 +494,8 @@ describe('Sanitation of Reported Data', () => {
       assert.equal(events[2].test.status, 'fail');
       assert.include(events[2].test.results.general.results[0].message.toLowerCase(), 'fail');
     });
-    it('emitted test data results contain all regular sections', () => assert.deepEqual(Object.keys(events[2].test.results), ['general', 'headers', 'body', 'statusCode']));
+    it('emitted test data results contain all regular sections', () => assert.hasAllKeys(events[2
+    ].test.results, ['general', 'headers', 'body', 'statusCode']));
```

Comment:
Strange newline, but given the test is decaffeinated and not humanly formatted, this is a minor issue.

Reply:
Fixed. Thanks :)